core/sqf/sql/scripts/dcscheck

```diff
@@ -171,17 +174,60 @@ if ( [ $sq_stat == 0 ] || [ $sq_stat == 1 ] ); then
    fi
    echo
   fi
-  echo -e "Process\t\tConfigured\tActual\t\tDown"
-  echo -e "---------\t----------\t------\t\t----"
-  echo -e "DcsMaster\t$cfg_dcsmaster_cnt\t\t$actual_dcsmaster_cnt\t\t$down_dcsmaster_cnt"
-  echo -e "DcsServer\t$cfg_dcsserver_cnt\t\t$actual_dcsserver_cnt\t\t$down_dcsserver_cnt"
-  echo -e "mxosrvr\t\t$cfg_mxo_cnt\t\t$act_mxo_cnt\t\t$down_mxo_cnt\n"
  else
    echo "DCS is not installed. Please install and configure DCS..."
    exit 1
  fi
 else
  echo "Trafodion is not started or is not operational..."
  echo
+ exit 1
 fi
+
+case $OPT_VALUE in
+    -m)
+        if [ -e $SQ_PDSH ]; then
+            STAT_CMD="$SQ_PDSH -w $activeMaster $jstatcmd -gc "
+        else
+            STAT_CMD=" $jstatcmd -gc "
+        fi
+        J_STATUS=`$STAT_CMD $activeDcsPid `
+        EC=`echo $J_STATUS | sed -n 1p | awk '{ print $20 }'`
+        EU=`echo $J_STATUS | sed -n 1p | awk '{ print $21 }'`
+        OC=`echo $J_STATUS | sed -n 1p | awk '{ print $22 }'`
+        OU=`echo $J_STATUS | sed -n 1p | awk '{ print $23 }'`
+        PERCENT_EU=$(echo "scale=2;$EU/$EC "| bc)
+        PERCENT_OU=$(echo "scale=2;$OU/$OC "| bc)
+
+        echo -e "Active DCSMaster VM status:"
+        echo -e "EC\tEU\tOC\tOU\tPERCENT_EU\tPERCENT_OU\t"
+        echo -e "$EC\t|$EU\t|$OC\t|$OU\t|$PERCENT_EU\t\t|$PERCENT_OU\t"
```

Comment:
Would be good to expand the title headers as indicated before  EC , OC and so on...  Please delete the debug lines

Reply:
I think there is no need to expand the title headers because jdk command 'jstat' behaves as the same way.